packages/frontend/src/components/navbar.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+
+import Link from "next/link";
+import { usePathname } from 'next/navigation';
+
+
+export default function Navbar(){
+    const pathname = usePathname();
+    return(
+        
+        <div style={{display: "flex", alignItems: "center", padding: "10px", marginLeft: "18px"}}>
```

Comment:
Ideally would be using tailwind for consistency across the codebase, but I leave it up to you